### PR TITLE
Assignee popup and add Vuetify

### LINKED ISSUE #68: Create component for searching member from a dropdown list of members.

### PR DESCRIPTION
**Pull request by pallabez**
### Issue

Closes #68

### What is the change?

Provide a small description in points of what did you change

##### Example :

- Done this
- Made these changes

### Is Development Tested?

- [ ] Yes
- [ ] No

### Before / After Change Screenshots

For visual or interaction changes. Please provide a video / screenshot.

##### Example:

Before :
Image/Video

After :
Image/Video

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: website/test/unit/store/users.test.js [hunk 1]
```diff
@@ -29,15 +29,13 @@ describe('Store/users', () => {
...
     let commit
-    
     beforeEach(() => {
       commit = spy()
     })
 
     afterEach(() => {
       restore()
     })
-
```
Comment (aakankshadhurandhar) [inline website/test/unit/store/users.test.js:40]: @pallabez This blank line is not needed.

Comment (pallabez) [reply website/test/unit/store/users.test.js:40]: Got it

**Comment by yesyash** [comment]
@pallabez are there any more changes to be made?
@aakankshadhurandhar Should I approve this PR in its current state?

**Comment by pallabez** [comment]
Merging this for now, do let me know if there are any other issue you might see in future.

**Pull request merged**